integreat_compass/cms/views/index.py

```diff
@@ -1,9 +1,125 @@
+from ast import literal_eval
+
+from django.shortcuts import render
 from django.views.generic import TemplateView
 
+from ..constants import offer_group_types, offer_mode_types
+from ..models import Offer, Tag
+
 
 class IndexView(TemplateView):
     """
-    Start page of the project
+    View to see all offers
     """
 
     template_name = "index.html"
+
+    def get(self, request, *args, **kwargs):
```

Comment:
If you don't intercept the rendering process itself, but just add additional context data, please just override `get_context_data()` instead of `get()`.

Reply:
I'm still on it

Reply:
I changed this function to the get_context_data function, but later added the get function for the filter form